product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -75,7 +75,7 @@ Tables are created across both groups, so any change goes to all nodes, not just
 
 One node in each group is selected as the group's write leader. The write leader is the target for the main application's writes and queries. All other nodes are described as shadow nodes (or "read-write replica"), waiting to take over when needed. If a node loses contact, processing switches immediately to a shadow node to continue processing. If a group fails, processing can switch to the other group. Scalability isn't the goal of this architecture.
 
-Since writes are mainly to only one node, the possibility of contention between nodes is reduced to almost zero. As a result, performance impact is much reduced.
+Since writes are mainly to only one node, the possibility of contention between nodes is reduced to almost zero. As a result, performance impact is significantly reduced.
```

Comment:
Greatly... people read significantly as indicating magnitudes

Reply:
got it. 👍